vizro-core/docs/pages/user-guides/aggrid.md

```diff
@@ -0,0 +1,281 @@
+# How to use AGGrids
+
+This guide shows you how to use the [AG Grid](https://www.ag-grid.com/) to visualize your tabular data in the dashboard.
+It is an interactive table/grid component designed for viewing, editing, and exploring large datasets.
+and Vizro's recommended table implementation.
+
+The Vizro [`AGGrid`][vizro.models.AGGrid] model is based on the [Dash AG Grid](https://dash.plotly.com/dash-ag-grid), which is in turn based the
```

Comment:
I'd put this paragraph right at the start.

Reply:
Hm, usually we have this one paragraph entry point as to what the guide will teach you, a little bit following: https://diataxis.fr/how-to-guides/#the-language-of-how-to-guides

Do you think it's ok to stick to that and then have this additional implementation info?

Besides that, to @antonymilne point, I will probably have a wider introduction about tables which will in turn be at the very top, so let's re-evaluate then